MaterialLib/PorousMedium/UnsaturatedProperty/CapillaryPressure/VanGenuchtenCapillaryPressureSaturation.cpp

```diff
@@ -77,6 +76,11 @@ double VanGenuchtenCapillaryPressureSaturation::getdPcdS(
 
         return -getdPcdSvGBar(1 - _saturation_r);
     }
+    if (saturation < _saturation_r)
+        return 0;
```

Comment:
Is `_saturation_r` a lower bound? Then I would expect to set the saturation to `_saturation_r`.

Reply:
This one is indeed correct: there is a cut-off in the function for s < s_r, so the derivative (this function) is 0.